pytest_invenio/fixtures.py

```diff
@@ -56,12 +56,14 @@ def instance_path():
     removed.
     """
     path = tempfile.mkdtemp()
+    env_instance_path = os.environ.get('INSTANCE_PATH', path)
```

Comment:
Should it not be:
`env_instance_path = os.environ.get('INVENIO_INSTANCE_PATH', path)` ???